samples/client/petstore-security-test/csharp/SwaggerClient/src/IO.Swagger/Client/JsonSubTypes.cs

```diff
@@ -0,0 +1,281 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace JsonSubTypes
+{
+    //  Copyright 2017 Emmanuel Counasse
```

Comment:
@manuc66 Can we remove the copyright note and license here? The auto-generated code in any languages should be unlicensed by default.